src/resources/Resource.js

```diff
@@ -11,12 +11,20 @@ export default class Resource {
     return url
   }
 
-  async request ({ path, fields = [], params = {}, method = 'get' }) {
+  async request ({
+    path,
+    fields = [],
+    params = {},
+    method = 'get',
+    mode = 'cors',
+    credentials = 'omit'
+  }) {
     const options = {
+      mode,
+      credentials,
       method: method.toLowerCase(),
```

Comment:
`fetch` automatically runs `toLowerCase()` on all header props, so this can be removed